files/fr/web/css/@font-face/line-gap-override/index.html

```diff
@@ -0,0 +1,74 @@
+---
+title: line-gap-override
+slug: Web/CSS/@font-face/line-gap-override
+browser-compat: css.at-rules.font-face.line-gap-override
+translation_of: 'Web/CSS/@font-face/line-gap-override'
+---
+<p>{{CSSRef}}</p>
+
+<p>Le descripteur CSS <strong><code>line-gap-override</code></strong> définit la mesure de l'écart de lignes de la police. La mesure de l'écart de lignes peut être l'écart recommandé par la police ou encore un écart externe.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">line-gap-override: normal;
+line-gap-override: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>normal</code></dt>
+  <dd>La valeur par défaut. Lorsqu'elle est utilisée, la valeur de la mesure est obtenue à partir du fichier de police.</dd>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}.</dd>
```

Comment:
https://github.com/mdn/translated-content/pull/1752/files#r678013156 here we go again :)

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1754/commits/8c3d7539226c6087156fd568d3a1e63194a4d060 :)